backend/api/Services/InspectionFindingsService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Api.Database.Context;
+using Api.Database.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Api.Services
+{
+    public class IInspectionFindingService(FlotillaDbContext context)
+    {
+        public async Task<List<InspectionFinding>> RetrieveInspectionFindings(TimeSpan timeSpan)
```

Comment:
Maybe better if this function takes `lastReportingTime` as a parameter instead of TimeSpan? Then the Handler can calculate the lastReportingTime

Reply:
Done.